src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -40,6 +40,8 @@
 import write_host_file
 # pylint: enable=wrong-import-position,import-error
 
+CLIENT_LOG = "/tmp/client.log"
```

Comment:
```suggestion
CLIENT_LOG = "/tmp/client_daos.log"
```

So that it gets collected and artifacted by the existing logic.